### DISCUSSION TITLE
Sometimes theia will loading forever

### DISCUSSION THREAD

**Discussion by lqd1434** [eclipse-theia/theia#13868] [Q&A] [answered]
theia version: 1.45.1
Sometimes when i start theia, theia will loading forever
and i noticed the last log like
`
connect 1yw1ueyhsdjsfskf2892wq423rsf
`

**Comment by JonasHelming** [comment]
Could you retest this with the current Theia version, please?

**Comment by lqd1434** [comment, marked as answer]
Thanks for your answer. I haven't encountered this problem again for the time being. Maybe we should consider upgrading to the latest community stable version of theia to improve stability and get bug fixes.
